cmd/search/query.go

```diff
@@ -15,8 +15,8 @@ func newQueryCmd() *cobra.Command {
 	var folderPath string
 	cmd := &cobra.Command{
 		Use:     "query",
-		Short:   "Manually Search using a specific Query and Indicies",
-		Long:    "\nRun a search query in IdentityNow using a specific Query and Indicies\n\n",
+		Short:   "Manually search using a specific query and indices",
```

Comment:
Part of wanting to break out the markdown separate from the actual go code was a desire to make it easier for people like yourself to go in and make docs updates and changes.

And yet here you are going in anyway, finding all the right places, and making incredibly helpful updates directly to the cobra command structs.

@james-haytko-sp you are truly a legend, and I'm so glad you are on our team.

Reply:
Thanks Luke! I appreciate it